src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/ColorPickerViewModel.cs

```diff
@@ -121,8 +127,13 @@ public int CopiedColorRepresentationIndex
 
         private void NotifySettingsChanged()
         {
+            // Using InvariantCulture as this is an IPC message
             SendConfigMSG(
-                   string.Format("{{ \"powertoys\": {{ \"{0}\": {1} }} }}", ColorPickerSettings.ModuleName, JsonSerializer.Serialize(_colorPickerSettings)));
+                   string.Format(
+                       CultureInfo.InvariantCulture,
```

Comment:
As mentioned previously this is probably CultureInfo.Ordinal.

Reply:
So for CultureInfo it looks like the only options are to give it a specific culture or to use InvariantCulture. The app uses it for string formatting and parsing, and from looking [here](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture?view=netcore-3.1) and [here](https://stackoverflow.com/questions/9760237/what-does-cultureinfo-invariantculture-mean) it looks like InvariantCulture is the best option since all the ones affected by this PR are internally-processed strings.

Reply:
Sorry good call.  My mistake.  Got a little carried away with the ordinal comments :)